doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -566,6 +569,11 @@ The key can then be moved to an external host and imported to GPG keyring with t
 Automated Certificate Management Environment (ACME) protocol enables automated certificate management of the authentication and sign
 certificates on the Security Server. More information about the required configuration is available in the [Security Server User Guide](ug-ss_x-road_6_security_server_user_guide.md#24-configuring-acme).
 
+### 3.7 Enabling EC keys for authentication and signing certificates
```

Comment:
This section is missing from the Table of Contents.

Also, please update the heading:

> 3.7 Enabling EC Keys for Authentication and Signing Certificates